index.html

```diff
@@ -28,7 +28,7 @@ <h1 class="title">2048</h1>
     </div>
 
     <div class="above-game">
-      <p class="game-intro">Join tiles and destroy <strong>EMPIRE!</strong></p>
+      <p class="game-intro">Join tiles and destroy the <strong>EMPIRE!</strong></p>
```

Comment:
These is only placeholder text since is replaced by localization, we should edit [js/localization.js](https://github.com/0x0800/2048-STARWARS/blob/gh-pages/js/localization.js).